src/core/types/addons.js

```diff
@@ -167,3 +167,7 @@ export type AddonType = {
   isWebExtension: boolean,
   themeData?: ThemeData,
 };
+
+export type CreateInternalAddonOptions = {|
+  _config?: Object,
```

Comment:
This should be `_config: typeof config` which is the same as `Object` but it helps to make the association.

Reply:
Hehe, I stole this from `core/reducers/redirectTo.js#21`.